lib/src/list.dart

```diff
@@ -326,3 +332,18 @@ class ListNotificationsController<T extends Object?> extends NotificationsContro
     streamController.addError(error);
   }
 }
+
+extension ListExtension<T> on List<T> {
+  /// Move the element at [from] to [to].
+  void move(int from, int to) {
```

Comment:
should we rename to `fromIndex` `toIndex` and also the other one that is oldIndex to be `fromIndex` as well

Reply:
here https://github.com/realm/realm-dart/pull/1037/files#diff-847b6223dda3c6c9a63c5b47a2dcc1b983c5035fc0cd6ccc152286c5c8449ef7R149

Reply:
or maybe use `from` and `to` everywhere since the API help says these are `indexes`

Reply:
Done